website/src/components/canvas/CanvasView.jsx

```diff
@@ -40,13 +45,21 @@ class CanvasView extends React.Component {
    */
   componentDidMount() {
     firebase.database().ref('/test').once('value').then((elemListSnap) => {
-      this.props.dispatch(initPositions(elemListSnap.val()));
+      this.props.dispatch(initElements(elemListSnap.val()));
     });
 
     firebase.database().ref('/test').on('child_changed', (elemSnap) => {
```

Comment:
Also, redux still fires on updates from fb that are from the current client, so you'd get 4 dispatches every time you try and change anything